README.md

```diff
@@ -31,59 +31,73 @@ To learn more about the future project direction, please check the [roadmap](.gi
 - [Roadmap](.github/ROADMAP.md)
 - [API Design Guidelines](.github/API_DESIGN.md)
 
-## Quickstart
+## Installation
+
+To install the latest official release:
+
+```
+pip install keras-cv tensorflow --upgrade
```

Comment:
Probably we could suggest a `venv` as we are implicitly breaking previous TensorFlow  and Keras installations with this command.

I understand that `--upgrade` is required to pass the runtime check over per-existing TF and Keras installation:
https://github.com/keras-team/keras-cv/blob/5227e52147343232a2183fefec1d31ed4c655079/keras_cv/version_check.py#L21-L31

Reply:
That's good advice. Personally I think it's common knowledge at this point that some containerization is required to have a good development and deployment experience in Python, so I don't think we're very special in this regard.